<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network</class>
 <widget class="QDialog" name="Network">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Networking Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/bullet_wrench.png</normaloff>:/icons/bullet_wrench.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable automatic router port forwarding (UPnP)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Manually specify port to listen on</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>63</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1025</number>
    </property>
    <property name="maximum">
     <number>49521</number>
    </property>
    <property name="value">
     <number>4950</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>100</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
